src/test/java/flyingnugget/ParserTest.java

```diff
@@ -0,0 +1,26 @@
+package flyingnugget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import flyingnugget.parser.Parser;
+import flyingnugget.storage.Storage;
+import flyingnugget.task.TaskList;
+
+public class ParserTest {
+
+    @Test
+    public void testTodo() {
```

Comment:
Similarly, do follow the convention `featureUnderTest_testScenario_expectedBehavior` so that the purpose of the test is more clear.